lightningd/pay.c

```diff
@@ -793,14 +793,6 @@ send_payment_core(struct lightningd *ld,
 	/* Now, do we already have one or more payments? */
 	payments = wallet_payment_list(tmpctx, ld->wallet, rhash);
 	for (size_t i = 0; i < tal_count(payments); i++) {
-		log_debug(ld->log, "Payment %zu/%zu: %s %s",
```

Comment:
Could instead put payment parts into bins of COMPLETE/PENDING/FAILED, then print something like "249 payments: COMPLETE 0 0msat, PENDING: 60 60msat, FAILED: 189 189msat"

Reply:
Removed this change for now, since it broke at least one test :-)
Will reconsider removing it again later.